internal/pkg/daemon/bpfrecorder/bpfrecorder.go

```diff
@@ -917,7 +913,13 @@ func (b *BpfRecorder) handleAppArmorEvents(apparmorEvents chan []byte) {
 		case uint8(probeTypeCap):
 			b.handleAppArmorCapabilityEvents(&apparmorEvent)
 		case uint8(probeTypeExit):
-			b.lockAppArmorRecording.Unlock()
+			d, _ := b.recordedExits.LoadOrStore(apparmorEvent.Pid, make(chan bool))
```

Comment:
Is it possible that this might lead to double closing the channel, for instance if the channel already exist in the map? 

I believe is just one exit event coming, but maybe make senses to check this just to be sure.